docs/publish.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Publishes docs changes to the gh-pages branch.
+
+# abort script at first error:
+set -e
```

Comment:
For maximum defensiveness, maybe `set -eu`? I always add `-o pipefail` reflexively too, but it's not strictly necessary here since there are no pipes.